From 7937cf205f653eb056588861e35489c4153c5b1c Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Fri, 1 Nov 2024 18:31:57 +0000
Subject: [PATCH 2/6] stress-fp-error: remove duplicated sqrt

The sqrt() is duplictated, remove it. Kudos to Woodrow Shen for spotting
this.

Closes: https://github.com/ColinIanKing/stress-ng/issues/450

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
---
 stress-fp-error.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/stress-fp-error.c b/stress-fp-error.c
index d5eac54c7b95..fe4d80146652 100644
--- a/stress-fp-error.c
+++ b/stress-fp-error.c
@@ -163,11 +163,6 @@ static int stress_fp_error(stress_args_t *args)
 		stress_fp_check(args, "sqrt(-1.0)", sqrt(-1.0), (double)NAN,
 			true, false, EDOM, FE_INVALID, &rc);
 #endif
-#if defined(EDOM) && defined(FE_INVALID)
-		stress_fp_clear_error();
-		stress_fp_check(args, "sqrt(-1.0)", sqrt(-1.0), (double)NAN,
-			true, false, EDOM, FE_INVALID, &rc);
-#endif
 #if defined(FE_INEXACT)
 #if !defined(STRESS_ARCH_ALPHA)
 		/*
-- 
2.47.0

